login.js

```diff
@@ -1,23 +1,228 @@
 const emailInput = document.getElementById("formInputEmail");
 const passwordInput = document.getElementById("formInputPassword");
 const loginButton = document.getElementById("loginButton");
+const togglePw = document.querySelector(".form-field-passVis");
+const togglePwImg = document.getElementById("passVisImg");
+const errorBorder = document.getElementsByClassName("error-border");
+const errorText = document.getElementsByClassName("error-text");
+//error-border[0] - email
+//error-border[1] - pw
 
-function checkEmailPwValidity() {
-  if (emailInput.checkValidity() && passwordInput.checkValidity()) {
-    console.log("둘다유효");
+const modal = document.getElementById("loginModal");
+const modalCloseBtn = document.getElementById("modalCloseBtn");
+const modalMsg = modal.querySelector(".modal-message");
+
+//모달 띄우기
+function showModal(message) {
+  modalMsg.textContent = message;
+  modal.classList.remove("hidden");
+}
+//모달 닫기
+modalCloseBtn.addEventListener("click", () => {
+  modal.classList.add("hidden");
+  if (loginCondition) {
+    window.location.href = "index.html"; //추후 items.html로 이동
+  }
+});
+
+const USER_DATA = [
+  { email: "codeit1@codeit.com", password: "codeit101!" },
+  { email: "codeit2@codeit.com", password: "codeit202!" },
+  { email: "codeit3@codeit.com", password: "codeit303!" },
+  { email: "codeit4@codeit.com", password: "codeit404!" },
+  { email: "codeit5@codeit.com", password: "codeit505!" },
+  { email: "codeit6@codeit.com", password: "codeit606!" },
+];
+
+function checkLoginInfo() {
+  const emailValue = emailInput.value;
+  const pwValue = passwordInput.value;
+  const loginValidity = USER_DATA.find((el) => {
+    return el.email == emailValue && el.password == pwValue;
+  });
```

Comment:
```suggestion
  const user = USER_DATA.find((el) => {
    return el.email == emailValue && el.password == pwValue;
  });
```

user 또는 exist 같은걸로 하는게 조금 더 좋아보입니다.